disco.py

```diff
@@ -1429,7 +1430,7 @@ def cond_fn(x, t, y=None):
                   skip_timesteps=skip_steps,
                   init_image=init,
                   randomize_class=randomize_class,
-                  eta=eta,
+                  eta=args.eta[1000-t_int] if type(args.eta) is np.ndarray else args.eta,
```

Comment:
really like this feature, can you provide a screenshot how it looks when this is intergrated?

Reply:
Here's the results: https://imgur.com/a/bVGj5Lc

And here's the link to the custom notebook with this feature implemented:
https://colab.research.google.com/drive/173sJy8VmXM8d9mUhjQcnEFr8WckpXIsc?usp=sharing